presto-base-jdbc/src/test/java/com/facebook/presto/plugin/jdbc/TestJdbcClient.java

```diff
@@ -110,4 +117,43 @@ public void testMetadataWithFloatAndDoubleCol()
                 new JdbcColumnHandle(CONNECTOR_ID, "COL3", JDBC_DOUBLE, DOUBLE),
                 new JdbcColumnHandle(CONNECTOR_ID, "COL4", JDBC_REAL, REAL)));
     }
+
+    @Test
+    public void testCreateTable()
+    {
+        String tableName = randomUUID().toString().toUpperCase(Locale.getDefault());
+        SchemaTableName schemaTableName = new SchemaTableName("exa_ple", tableName);
+        List<ColumnMetadata> expectedColumns = ImmutableList.of(new ColumnMetadata("columnA", BigintType.BIGINT, null, null, false));
+
+        jdbcClient.createTable(new ConnectorTableMetadata(schemaTableName, expectedColumns));
+
+        JdbcTableHandle tableHandle = jdbcClient.getTableHandle(schemaTableName);
+        assertEquals(tableHandle.getTableName(), tableName);
+        assertEquals(jdbcClient.getColumns(session, tableHandle), ImmutableList.of(new JdbcColumnHandle(CONNECTOR_ID, "COLUMNA", JDBC_BIGINT, BigintType.BIGINT)));
+    }
+
+    @Test
+    public void testAlterColumns()
+    {
+        String tableName = randomUUID().toString().toUpperCase(Locale.getDefault());
+        SchemaTableName schemaTableName = new SchemaTableName("exa_ple", tableName);
+        List<ColumnMetadata> expectedColumns = ImmutableList.of(
+                new ColumnMetadata("columnA", BigintType.BIGINT, null, null, false));
+
+        jdbcClient.createTable(new ConnectorTableMetadata(schemaTableName, expectedColumns));
+
+        JdbcTableHandle tableHandle = jdbcClient.getTableHandle(schemaTableName);
+        assertEquals(tableHandle.getTableName(), tableName);
+        assertEquals(jdbcClient.getColumns(session, tableHandle), ImmutableList.of(
+                new JdbcColumnHandle(CONNECTOR_ID, "COLUMNA", JDBC_BIGINT, BigintType.BIGINT)));
+
+        jdbcClient.addColumn(tableHandle, new ColumnMetadata("columnB", BigintType.BIGINT, null, null, false));
```

Comment:
Use a different data type than the first column, so it's clear they aren't getting mixed up somehow